pypackage/{{ cookiecutter.slug }}/src/{{ cookiecutter.package_name }}/cli.py

```diff
@@ -1,9 +1,5 @@
 from argparse import ArgumentParser
-
-try:
-    from importlib.metadata import version
-except ModuleNotFoundError:
-    from importlib_metadata import version
+from importlib.metadata import version
```

Comment:
`importlib_metadata` is no longer needed now that I've dropped support for Python 3.7